assets/The Boston DAO.md

```diff
@@ -39,7 +39,9 @@ c. Open, transparent, and adaptable
 >i. We believe that the future of governance involves operating publicly and deliberately. Rather than assuming an uneducated majority, we seek to educate the majority. 
 
 d. Inclusive and diverse
->i. The Boston DAO is holistically meritocratic — understanding that every decision is not black and white, but also pushing to innovate and govern in a truly digital world.
```

Comment:
I feel like the current text written under this bullet really has very little to do with diversity and inclusion, and is generally a very weak statement.  I'd really like us to make a stronger stance around our commitment to DE&I as I think this is an area where the broader web3 space is severely lacking and where we can do a lot to lead by example